### PR TITLE
supeer pull request

### PR DESCRIPTION
**Pull request by yuzhangClaremont**
plz peer review @YucWu

### CODE REVIEW

File: problems.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#TODO: fix this whole fuction
```
Comment (YucWu) [inline problems.rb:1]: OK. I will check on that.

File: problems.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+def hithisfuncissocomp!!licated(n)
```
Comment (YucWu) [inline problems.rb]: I love what you're doing here but it seems that the name of the function is a bit too long and not so intuitive. We could replace it with something like sayHiFourTimes(n).

File: problems.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    for i in 0..n
```
Comment (YucWu) [inline problems.rb]: This loop is from 0 to n, which means we are looping for n+1 times, which is 1 more than we need. Better still, we could do a for loop from 0 to 4n-1, which solves the whole issue.

File: problems.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+        for j in 0..3
+            puts 'HI'
```
Comment (YucWu) [inline problems.rb]: This definitely works but it would better store “Hi” in a variable to avoid magic values.

File: problems.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+        end
+    end
+
+end
```
Comment (yuzhangClaremont) [inline problems.rb]: need last line

**Review by yuzhangClaremont — COMMENTED** [review]
test test

**Review by YucWu — CHANGES_REQUESTED** [review]
Finished review

**Review by YucWu — APPROVED** [review]
Great!!

**Pull request merged**